external-import/flashpoint/src/flashpoint.py

```diff
@@ -356,68 +351,30 @@ def _convert_to_markdown(self, content):
 
     def _import_apt(self, work_id):
         # Query params
-        url = self.flashpoint_api_app_url + "/documents/apt/wiki"
-        headers = {
-            "Content-Type": "application/json",
-            "Authorization": "Bearer " + self.flashpoint_api_key,
-        }
-        response = requests.get(url, headers=headers)
-        data = json.loads(response.content)
-        objects = []
-        try:
-            if "data" in data:
-                for apt in data["data"]:
-                    intrusion_set_stix = stix2.IntrusionSet(
-                        id=IntrusionSet.generate_id(apt["apt_group"]),
-                        name=apt["apt_group"],
-                        aliases=apt["aliases"],
-                        description=self._convert_to_markdown(apt["body"]["raw"]),
-                        created_by_ref=self.identity["standard_id"],
-                        object_marking_refs=[stix2.TLP_GREEN.get("id")],
-                    )
-                    objects.append(intrusion_set_stix)
-            self.helper.send_stix2_bundle(
-                stix2.Bundle(
-                    objects=objects,
-                    allow_custom=True,
-                ).serialize(),
-                work_id=work_id,
-            )
-        except Exception as e:
-            self.helper.log_error(str(e))
+        msg = (
+            "/document/malware/wiki has been deprecated by Flashpoint Ignite, the option import malware does not "
+            "work anymore and will be removed in a future version."
+        )
+        warnings.warn(
+            message=msg,
+            category=DeprecationWarning,
+            stacklevel=2,
+        )
+        self.helper.connector_logger.warning(msg)  # warns connector user
+        self.helper.api.work.to_processed(work_id, msg)  # warns OpenCTI user
```

Comment:
If you use to_processed() to close the job, it will appear as ‘complete’ on the frontend. However, other imports such as import_report or import_community based on the same work_id will still be in progress. This can give the impression of a ‘completed’ job when the import is still running in the background. Otherwise the very cool warning!